java/memory/src/main/java/org/apache/arrow/memory/AllocationManager.java

```diff
@@ -35,6 +35,7 @@
 import io.netty.buffer.ArrowBuf;
 import io.netty.buffer.PooledByteBufAllocatorL;
 import io.netty.buffer.UnsafeDirectLittleEndian;
+import sun.misc.Unsafe;
```

Comment:
why added?

Reply:
My bad :( result of auto imports based on some code changes that were later removed.